Cargo.toml

```diff
@@ -19,7 +19,7 @@ ark-ff = { version = "0.4.0", optional = true }
 ark-serialize = { version = "0.4.2" }
 ark-std = { version = "0.4.0" }
 blitzar-sys = { version = "1.15.1" }
-curve25519-dalek = { version = "3", features = ["serde"] }
+curve25519-dalek = { version = "4", features = ["serde", "rand_core"] }
```

Comment:
I think `rand_core` is only used for testing, correct?
If that is the case, could you add `rand_core` to the `dev-dependencies` instead of here.

Reply:
You are correct. Added, thank you.